src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -795,7 +795,7 @@ internal DefinitionResult GetDefinition(TextDocumentPosition textDocumentPositio
             }
 
             // Get token from selected text
-            Token selectedToken = ScriptDocumentInfo.GetToken(scriptParseInfo, textDocumentPosition.Position.Line + 1, textDocumentPosition.Position.Character);
+            Token selectedToken = ScriptDocumentInfo.GetPeakDefinitionToken(scriptParseInfo, textDocumentPosition.Position.Line + 1, textDocumentPosition.Position.Character + 1);
```

Comment:
`Peek` instead of `Peak
![Gold Peak Tea](https://i5.walmartimages.com/asr/bef929b1-7c24-4a7b-8882-fbea8069d8fc_1.0bd9f5be30922509b9ac5c4a4514eb95.jpeg?odnHeight=450&odnWidth=450&odnBg=FFFFFF)